src/controllers/user.controller.js

```diff
@@ -153,7 +153,7 @@ class UserController {
       const token = await Helper.generateToken(payload);
       return res.status(201).json({
         status: 201,
-        message: 'successfully created account ',
+        message: 'Your account has been successfully created. An email has been sent to you with detailed instructions on how to activate it.',
```

Comment:
The message seems to be very long,I suggest "Your account has been successfully created, activation mail sent"